net/sslh/src/opnsense/service/conf/actions.d/actions_sslh.conf

```diff
@@ -0,0 +1,31 @@
+################################################################################
+# Service Actions                                                              #
+################################################################################
+
+[status]
+command:/usr/local/etc/rc.d/sslh onestatus || exit 0
+parameters:
+type:script_output
+message: sslh: requesting status
+description:
+
+[start]
+command:/usr/local/etc/rc.d/sslh onestart || exit 0
+parameters:
+type:script
+message: sslh: starting
+description: sslh: Start service
+
+[stop]
+command:/usr/local/etc/rc.d/sslh onestop || exit 0
```

Comment:
|| exit 0 shouldn't be used for anything but status where unfortunately we get an error when the service is not running which is not really an error if we stopped it / don't have it configured.